src/Gui/Navigation/NavigationStyle.cpp

```diff
@@ -61,12 +61,13 @@ class FCSphereSheetProjector : public SbSphereSheetProjector {
     using inherited = SbSphereSheetProjector;
 
 public:
-    enum OrbitStyle {
-        Turntable,
+    enum OrbitStyle
+    {
+        RoundedArcball,
         Trackball,
-        FreeTurntable,
         TrackballClassic,
-        RoundedArcball
+        Turntable,
+        FreeTurntable,
     };
```

Comment:
We need to keep the order of this enum or peoples orbit style settings will be scrambled.

We only need to change the order in the ui